boards/msb-430/include/board.h

```diff
@@ -43,8 +43,8 @@ extern "C" {
 /**
  * @brief   Define the CPU model for the <msp430.h>
  */
-#ifndef __MSP430F1612__
-#define __MSP430F1612__
+#ifndef MSP430F1612
```

Comment:
> most of them are in fact correct

Do you mean most of my revisions here in this PR are correct?

But, as you have mentioned earlier:
> Just realized... Here and in the other MSP430 board: These aren't header guards.

What I have revised here in this PR are nearly all MSP430 series boards which seem to have MSP430 CPU.
So, it seems contradictory to me. :-)